src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -794,20 +797,18 @@ public enum LegendsAwakened implements LogicCardInfo {
           weakness P, '+20'
           pokePower "Time Walk", {
             text "Once during your turn, when you put Azelf from your hand onto your Bench, you may look at all of your face-down Prize cards. If you do, you may choose 1 Pokémon you find there, show it to your opponent, and put it into your hand. Then, choose 1 card in your hand and put it as a Prize card face down."
-            actionA {
-              assert my.hand : "No cards in hand"
-              checkLastTurn()
-              powerUsed()
-
-              def newPrize = my.hand.select(hidden: true, "Card to put into Prizes").first()
-
-              def tar = my.prizeCardSet.faceDownCards.select(hidden: false, "Choose a Prize card to replace with one in your hand.").first()
-              my.hand.add(tar)
-
-              def indexOfOldPrize = my.prizeCardSet.indexOf(tar)
-              my.prizeCardSet.set(indexOfOldPrize, newPrize)
-              my.prizeCardSet.setVisible(newPrize, true)
-              my.hand.remove(newPrize)
+            onActivate {r->
+              if (r==PLAY_FROM_HAND && confirm("Use Time Walk?")){
+                assert my.hand : "No cards in hand"
```

Comment:
```suggestion
```
> Q 自分の手札が0枚のとき、アグノムのポケパワー｢タイムウォーク｣を使って、ウラになっているサイドからポケモンをえらぶことはできますか?
A はい、できます。
ただし、サイドからえらんだポケモン以外に手札がないため、えらんだポケモンをもう一度サイドとしてもどして、ポケパワーを終わります。

Translation:
> Q When I have no cards in my hand, can I use Azelf's PokéPower "Time Walk" to select a Pokémon from my facedown Prize Cards?
A Yes, you can. However, since you don't have any other Pokémon [rather, cards] in your hand other than the one you selected from the Prize Cards, you have to return the selected Pokémon to your Prize Cards again to finish the PokéPower.